application/features/bootstrap/MfaContext.php

```diff
@@ -5,8 +5,10 @@
 use common\models\EmailLog;
 use common\models\Mfa;
 use common\models\MfaBackupcode;
+use common\models\MfaWebauthn;
 use common\models\User;
 use Webmozart\Assert\Assert;
+use yii\base\BaseObject;
```

Comment:
`BaseObject` doesn't seem to be used in this file. Is this `use` statement necessary?